qwen_agent/llm/openvino.py

```diff
@@ -0,0 +1,154 @@
+from typing import Dict, Iterator, List, Optional
+from threading import Thread
+
+from optimum.intel.openvino import OVModelForCausalLM
+from transformers import (AutoTokenizer, AutoConfig,
```

Comment:
We would like to make dependencies like `transformers` and `optimum` optional for users of qwen-agent, as these two can be quite heavy to install. 

If you have the time, would you kindly refactor the code to import these packages only where they are explicitly used, rather than at the top of the file? This would be greatly appreciated.

For example, move
```py
from optimum.intel.openvino import OVModelForCausalLM
```
to
```py
class OpenVINO(BaseTextChatModel):
    def __init__(self, cfg: Optional[Dict] = None):
        ...

        from optimum.intel.openvino import OVModelForCausalLM  # Move this import here!

        self.ov_model = OVModelForCausalLM.from_pretrained(
        ...
```

Reply:
@JianxinMa updated, pls help to review, thanks